features/platform_management.feature

```diff
@@ -0,0 +1,252 @@
+@api @management
+Feature: Platform management
+
+  When operating Probe Dock, we want to get some metrics on the database such the size of each table with their indexes.
+
+  There is also the possibility to get few metrics for an organization admin such the number of records
+  for the main objects:
+    - Payloads
+    - Projects
+    - Tests
+    - Results
+
+
+
+  Scenario: A Probe Dock administrator should retrieve the DB stats
+    Given user palpatine who is a Probe Dock admin exists
+    When palpatine sends a GET request to /api/platformManagement/dbStats
+    Then the response code should be 200
+    And the response body should be a JSON array with the following tables:
```

Comment:
I find the naming of this assertion confusing. It starts with "And the response body should be a JSON array...", leading me to believe that I'm going to see JSON in the text block below, but then it says "... with the following tables", and then I understand because we've discussed about it but I don't think anyone else would.

I think something like "And the response should contain DB stats for the following tables:" would be clearer.